.github/workflows/format-check.yml

```diff
@@ -0,0 +1,22 @@
+name: Format Check
+
+on: push
+
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+  NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+jobs:
+  format-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          persist-credentials: false
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: "npm"
+      - run: npm ci
+      - run: npm run format:check
```

Comment:
To save on pipeline time, I would do this in the lint job. They're both pretty fast, and both have to do with code structure.

Reply:
I was thinking of it as well! Ok then I will put them together ☺️

Reply:
Ok, done!